src/app/services/views/MeshExternalServiceListView.vue

```diff
@@ -13,179 +13,210 @@
       :render="false"
       :title="t(`services.routes.mesh-external-service-list-view.title`)"
     />
-    <AppView
-      :docs="t('services.mesh-external-service.href.docs')"
+    <DataSource
+      :src="uri(egressSources, `/zone-cps/:name/egresses`, {
+        name: '*',
+      }, {
+        page: 1,
+        size: 100,
+      })"
+      v-slot="{ data: egresses }"
     >
-      <KCard>
-        <DataLoader
-          :src="uri(sources, '/meshes/:mesh/mesh-external-services', {
-            mesh: route.params.mesh,
-          },{
-            page: route.params.page,
-            size: route.params.size,
-          })"
+      <template
+        v-for="warnings in [[
+          [t('services.mesh-external-service.notifications.mtls-warning'), typeof props.mesh.mtlsBackend === 'undefined'],
+          [t('services.mesh-external-service.notifications.no-zone-egress'), egresses && !egresses.items.find((egress) => typeof egress.zoneEgressInsight.connectedSubscription !== 'undefined')],
```

Comment:
This is the biggest addition. Just uses `[i18nText, conditionalWhichWhenTrueShowsTheI18nText]`. The rest is just renesting.